docs/api.md

```diff
@@ -966,43 +1175,45 @@ Example Response Body:
 {
   "balanceUpdates": [
     {
-      "uuid": "6c2f7a09-d602-4c1a-a435-e915ed64423d",
-      "owner": "0xba7353ff41853ca0429a594584ae256231decb51",
-      "createdAt": "2020-01-18T17:56:06.219Z",
-      "updatedAt": "2020-02-14T19:01:06.198Z",
+      "uuid": "35f3d8cc-22a9-447e-b0b6-051f3e7272b7",
+      "owner": "0x77a035b677d5a0900e4848ae885103cd49af9633",
       "market": "PBTC-USDC",
-      "deltaMargin": "0.00000000121",
-      "newMargin": "10.00000000001",
-      "deltaPosition": "0",
-      "newPosition": "15000",
-      "indexValue": "6000000000000000000000",
-      "indexTimestamp": "1585934124",
-      "orderNumber": "956855500050000",
+      "deltaMargin": "89957950",
+      "newMargin": "130127513",
+      "deltaPosition": "-1000000",
+      "newPosition": "-990000",
+      "indexValue": "1.204924898727124293",
+      "indexTimestamp": "1590089676",
+      "isPendingBlock": false,
+      "orderNumber": "1011116901510002",
+      "createdAt": "2020-05-21T19:36:03.270Z",
+      "updatedAt": "2020-05-21T19:36:03.270Z"
     }
   ]
 }
 ```
 
 Balance Update Response Object:
 
-| Field Name     | Description                                      |
-|----------------|--------------------------------------------------|
-| uuid           | The unique ID for the balance update.            |
-| owner          | The wallet address of the user.                  |
-| market         | The perpetual market, e.g. `PBTC-USDC`.          |
-| deltaMargin    | The change in settlement token (e.g. USDC).      |
-| newMargin      | The new balance of settlement token (e.g. USDC). |
-| deltaPosition  | The change in position token (e.g. PBTC).        |
-| newPosition    | The amount in position token (e.g. PBTC).        |
-| indexValue     | The new index value of the account.              |
-| indexTimestamp | The new index timestamp of the account.          |
-| orderNumber    | Number used for ordering the balance updates.    |
+| Field Name     | Description                                           |
+|----------------|-------------------------------------------------------|
+| uuid           | The unique ID for the balance update.                 |
+| owner          | The wallet address of the user.                       |
+| market         | The perpetual market, e.g. `PBTC-USDC`.               |
+| deltaMargin    | The change in settlement token (e.g. USDC).           |
+| newMargin      | The new balance of settlement token (e.g. USDC).      |
+| deltaPosition  | The change in position token (e.g. PBTC).             |
+| newPosition    | The amount in position token (e.g. PBTC).             |
+| indexValue     | The new index value of the account.                   |
+| indexTimestamp | The timestamp for when the index value was set.       |
+| orderNumber    | Number used for ordering the balance updates.         |
+| isPendingBlock | Whether the specific balance update is pending or not |
 
 
 ### GET `/v1/standard-actions`
```

Comment:
Is this a copy of the `GET /v1/standard-actions` section above? Since we had a lot of endpoints applicable to both Solo and Perpetual, I added the `## Trading Endpoints (Solo and Perpetual)` section above in a previous change. I wonder if it would make sense to put this there, since we don't usually document the same endpoint twice

Reply:
Yeah, it's the same endpoint. I documented it separately, because some of the fields for standard actions do not apply to both solo and perpetual. eg: we currently do not set the `pnl` field for perpetual. I could merge the two and have a separate explanation for both

Reply:
merged into a single description